.toys/new-library.rb

```diff
@@ -153,8 +158,9 @@ def gem_version_file
 end
 
 def add_fillers manifest
-  manifest.each_key do |key|
-    manifest["#{key}+FILLER"] = "0.0.0" unless key.end_with? "+FILLER"
```

Comment:
What's going on here?

Reply:
Getting the following error while generating a new library - `RuntimeError: can't add a new key into hash during iteration`

Reply:
Oh, I see, this was a Rubocop overreach. I think it used to be manifest.keys.each, which worked because it makes a copy of the keys array, but Rubocop made us change it to each_key which is subtly different. Okay, sounds good.

Reply:
Tried that but `manifest.keys.each` results in rubocop failure. This was the reason we shifted to `manifest.each_key`.